tests/fast-integration/test/fixtures/jaeger/jaeger.yaml

```diff
@@ -0,0 +1,307 @@
+    app.kubernetes.io/name: tracing-jaeger-query
+    app.kubernetes.io/part-of: jaeger
+  name: tracing-jaeger-query
+  namespace: kyma-system
+spec:
+  ports:
+  - name: http-query
+    port: 16686
+    protocol: TCP
+    targetPort: 16686
+  - name: grpc-query
+    port: 16685
+    protocol: TCP
+    targetPort: 16685
+  selector:
+    app: jaeger
+    app.kubernetes.io/component: all-in-one
+    app.kubernetes.io/instance: tracing-jaeger
+    app.kubernetes.io/managed-by: jaeger-operator
+    app.kubernetes.io/name: tracing-jaeger
+    app.kubernetes.io/part-of: jaeger
+
+---
+# Source: tracing/templates/kyma-additions/grafana-datasource.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tracing-jaeger-grafana-datasource
+  labels:
+    app.kubernetes.io/name: tracing
+    app.kubernetes.io/instance: jaeger
+    grafana_datasource: "1"
+data:
+    jaeger-datasource.yaml: |-
+      apiVersion: 1
+      datasources:
+      - name: Jaeger
+        type: jaeger
+        access: proxy
+        url: http://tracing-jaeger-query.kyma-system:16686
+        editable: true
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: jaeger
+    app.kubernetes.io/component: all-in-one
+    app.kubernetes.io/instance: tracing-jaeger
+    app.kubernetes.io/managed-by: jaeger-operator
+    app.kubernetes.io/name: tracing-jaeger
+    app.kubernetes.io/part-of: jaeger
+  name: tracing-jaeger
+  namespace: kyma-system
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: jaeger
+      app.kubernetes.io/component: all-in-one
+      app.kubernetes.io/instance: tracing-jaeger
+      app.kubernetes.io/managed-by: jaeger-operator
+      app.kubernetes.io/name: tracing-jaeger
+      app.kubernetes.io/part-of: jaeger
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        linkerd.io/inject: disabled
+        prometheus.io/port: "14269"
+        prometheus.io/scrape: "true"
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+      labels:
+        app: jaeger
+        app.kubernetes.io/component: all-in-one
+        app.kubernetes.io/instance: tracing-jaeger
+        app.kubernetes.io/managed-by: jaeger-operator
+        app.kubernetes.io/name: tracing-jaeger
+        app.kubernetes.io/part-of: jaeger
+    spec:
+      containers:
+      - args:
+        - --collector.otlp.enabled=true
+        - --log-level=info
+        - --memory.max-traces=10000
+        - --query.ui-config=/etc/config/ui.json
+        - --sampling.strategies-file=/etc/jaeger/sampling/sampling.json
+        env:
+        - name: SPAN_STORAGE_TYPE
+          value: memory
+        - name: METRICS_STORAGE_TYPE
+        - name: COLLECTOR_ZIPKIN_HOST_PORT
+          value: :9411
+        - name: JAEGER_DISABLED
+          value: "false"
+        image: eu.gcr.io/kyma-project/external/jaegertracing/all-in-one:1.37.0
```

Comment:
Maybe just switch to latest upstream version to remove any dependency to the old component